tests/test_completion.py

```diff
@@ -635,7 +635,7 @@ def test_tokens_for_completion_redirect(cmd2_app):
     endidx = len(line)
     begidx = endidx - len(text)
 
-    cmd2_app.allow_redirection = True
+    cmd2_app.statement_parser.allow_redirection = True
```

Comment:
Why not have these use `cmd2_app's` `allow_redirection` property instead?

Reply:
Currently the ``allow_redireciton`` property is read-only - i.e. it provides the automatic getter, but not a corresponding setter.  I did debate making this a read-write property that provides both a getter and a setter.  Do you think that would be a good idea?

Reply:
I added a setter for the ``allow_redirection`` property and modified the unit test to use it.